include/algorithms/boosting/boosting_model.h

```diff
@@ -57,7 +57,7 @@ namespace interface1
  * \snippet boosting/boosting_model.h Parameter source code
  */
 /* [Parameter source code] */
-struct DAAL_EXPORT Parameter : public classifier::Parameter
+struct DAAL_EXPORT Parameter : public classifier::interface1::Parameter
```

Comment:
Does it mean that this class is deprecated as there is no interface2::Parameter?

Reply:
yes, full boosting:: namespace is deprecated